finalPrep/annotations.js

```diff
@@ -2,41 +2,72 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+
+//Create a dynamic function with multiple (5) parameters
 function buildABear(name, age, fur, clothes, specialPower) {
+//Declare a variable (*greeting*) and assign a value that is a string containing the parameter *name* interpolated in.
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+//Declare a variable (*demographics) and assign the value to an array containing the (elements? parameters? not sure) *name* and *age*.
   var demographics = [name, age];
+//Declare a variable (*powerSaying*) and assign the value to a string containing the parameter *specialPower* concatenated in.
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+//Create an object (*buildBear*). Begin by using the keyword `var` followed by the object's name (*builtBear*).
   var builtBear = {
+//Add key (*basicInfo*) value (*demographics*) pair to object. Value is a varible used above.
     basicInfo: demographics,
+//Add another key value pair to object. Value is a parameter from above.
     clothes: clothes,
+//Add another key value pair to object. Value is a parameter from above.
     exterior: fur,
+//Add another key value pair to object. Value is assigned here.
```

Comment:
Nice- well done noting where your values come from. This is a valuable thing to track in code.